### PR TITLE
add mt classifier by using deepspeed

### PR DESCRIPTION
**Pull request by yuzhangogogo**
训练脚本示例：
deepspeed finetune/run_classifier_mt_deepspeed.py \
--spm_model_path models/spm.model \
--config_path models/llama2/7b_config.json  --deepspeed_config models/deepspeed_zero3_config.json \
--dataset_path_list ./train.tsv 
--epochs_num 3 --batch_size 48 --world_size 8 --scheduler cosine --warmup 0.03 --seq_length 300 \
--pooling last --deepspeed_checkpoint_activations --report_steps 2 --learning_rate 1e-5 \
--pretrained_model_path models_5000.bin \
--output_model_path models/output

推理脚本示例：
deepspeed inference/run_classifier_mt_deepspeed_infer.py \
--spm_model_path models/spm.model \
--config_path models/llama2/7b_config.json  --deepspeed_config models/deepspeed_zero3_config.json \
--load_model_path models/output/model.bin \
--test_path dev.tsv \
--labels_num_list 2 2 2 2 2 2 2 2 2 2 2 \
--prediction_path result.tsv \
--output_logits \
--output_prob \
--pooling last \
--world_size 8 \
--batch_size 10

### CODE REVIEW

**Comment by yuzhangogogo** [comment]
add mt classifier by using deepspeed
